src/renderers/dom/client/utils/setInnerHTML.js

```diff
@@ -28,7 +31,20 @@ var createMicrosoftUnsafeLocalFunction = require('createMicrosoftUnsafeLocalFunc
  */
 var setInnerHTML = createMicrosoftUnsafeLocalFunction(
   function(node, html) {
-    node.innerHTML = html;
+    if ('innerHTML' in node) {
```

Comment:
what would be the advantage of checking the namespace as opposed to just checking if the node is a `SVGElement `?